### DISCUSSION TITLE
Making the moods formatter example work

### DISCUSSION THREAD

**Discussion by EsraKlemann** [simeydotme/svelte-range-slider-pips#68] [Formatting / Styling] [answered]
Hi! I've been trying to make this example with the moods (emoji's) formatter to work. The given example makes repl say that values are not declared. If I change moods to 'values' it tells me it cannot bind to a non writable variable... 
I'm very inexperienced, so probably missing something here...

**Comment by simeydotme** [comment, marked as answer]
Hi Esra!

Thanks for starting a discussion on this!
May you please share what you have so far? [A REPL forked from this one](https://svelte.dev/repl/030797781fd64ad88302d1343f5b2c43) would be great, or feel free to provide your own.

We should be able to get you sorted

**Reply by simeydotme** [reply]
did you get this sorted @EsraKlemann ? 
Would you share your resolution if so :)

**Reply by EsraKlemann** [reply]
Hi there,

It's been a while since I worked on this, [this would be the simple version](https://svelte.dev/repl/34951ae251dd4d6c8303817eda37e4c4?version=3.48.0) of how I solved it. Might not have been the most elegant way... but it worked for me :)

**Reply by simeydotme** [reply]
looks great !
